src/libsystemd-network/dhcp6-option.c

```diff
@@ -206,6 +206,8 @@ bool dhcp6_option_can_request(uint16_t option) {
         case SD_DHCP6_OPTION_V6_DOTS_ADDRESS:
         case SD_DHCP6_OPTION_IPV6_ADDRESS_ANDSF:
                 return true;
+        case SD_DHCP6_OPTION_PRIVATE_BASE:
+        case SD_DHCP6_OPTION_PRIVATE_LAST:
```

Comment:
hmm, just testing the first and last is a bit weird. in particular as this is entirely redundant of course, because the `default` already covers this.

I think it would make sense to add:

```c
case _SD_DHCP6_OPTION_PRIVATE_BASE .. SD_DHCP6_OPTION_PRIVATE_LAST:
```

here though, simply to make explicit that the private options cannot be requested

Reply:
Fixed it.